dask/dataframe/utils.py

```diff
@@ -530,9 +531,11 @@ def assert_eq(
     check_dtype=True,
     check_divisions=True,
     check_index=True,
-    scheduler="sync",
+    scheduler=None,
     **kwargs,
 ):
+    if scheduler is None:
+        scheduler = config.get("dataframe.assert-eq.scheduler", "sync")
```

Comment:
Is this the normal way to set default values in the config?

Reply:
dask used to not have `pyyaml` as a dependency, so all config values needed to also exist in the code at the `get` calls (since the default values may not be loaded if `pyyaml` isn't installed). This is no longer the case, so specifying in the yaml file is preferable.

Reply:
Thanks for the context 🙂 addressed this in https://github.com/dask/dask/pull/8811/commits/686630c1eeb49564e7c7a4f7bad9595b28db2f46